hmrc-mongo-metrix-play-26/src/main/scala/uk/gov/hmrc/metrix/persistence/MongoMetricRepository.scala

```diff
@@ -27,8 +27,11 @@ import uk.gov.hmrc.mongo.play.PlayMongoCollection
 import scala.concurrent.{ExecutionContext, Future}
 
 class MongoMetricRepository(collectionName: String = "metrics", mongo: MongoComponent)(implicit ec: ExecutionContext)
-  extends PlayMongoCollection[PersistedMetric](collectionName = collectionName, mongoComponent = mongo, domainFormat = PersistedMetric.format,
-    indexes = Seq(
+  extends PlayMongoCollection[PersistedMetric](
+    collectionName = collectionName,
+    mongoComponent = mongo,
+    domainFormat   = PersistedMetric.format,
+    indexes        = Seq(
       IndexModel(ascending("name"), IndexOptions().name("metric_key_idx").unique(true).background(true))
```

Comment:
Know this isn't one of yours, but is it worth changing this to be camelCased for consistency with the other indexes that we have?